README.md

```diff
@@ -1,2 +1,47 @@
-# workflows
-Reusable workflows for use in the Fastify organization
+# Workflows
+
+Reusable workflows for use in the Fastify organization.
+
+## Intro
+
+GitHub [introduced reusable workflows](https://github.blog/2021-11-29-github-actions-reusable-workflows-is-generally-available/) on 2021-11-29 which, as the name suggests, are workflows that can be referenced across the entirety of GitHub.
+
+For more information, including limitations, [see the GitHub Docs](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows).
+
+## Usage
+
+A reusable workflow is called by using the `uses` keyword in another workflow:
+
+```yml
+name: CI
+
+on:
+    push:
+        branches:
+            - main
+            - master
+        paths-ignore:
+            - 'docs/**'
+            - '*.md'
+    pull_request:
+        branches:
+            - main
+            - master
+        paths-ignore:
+            - 'docs/**'
+            - '*.md'
+
+jobs:
+    call-reuseable-workflow:
```

Comment:
We should add an example on how to use it with an external source, like `mongodb`

Reply:
Do we know if this is possible to do?

Reply:
Not 100% on what you mean @Eomm , sorry!

Do you mean using services like what is done in some of the Fastify repos such as [fastify-postgres](https://github.com/fastify/fastify-postgres/blob/master/.github/workflows/ci.yml)?

Reply:
Yes, exactly!

I don't understand if this is working reading the docs

Reply:
From reading the docs, I have no idea either!

I imagine it would look like so, but can't confirm it would work.
I can open a test PR on fastify-postgres to see if it would work with my fork?

```yml
name: CI
on:
  push:
    paths-ignore:
      - 'docs/**'
      - '*.md'
  pull_request:
    paths-ignore:
      - 'docs/**'
      - '*.md'
jobs:
  call-reuseable-workflow:
    runs-on: ubuntu-latest
    services:
       postgres:
        image: postgres:11-alpine
        env:
            POSTGRES_USER: postgres
            POSTGRES_DB: postgres
            POSTGRES_PASSWORD: postgres
        ports:
            - '5432:5432'
        options: >-
            --health-cmd pg_isready --health-interval 10s --health-timeout 5s
            --health-retries 5
    steps:
        - uses: fastify/workflows/.github/workflows/plugins-ci.yml@v1
          with:
            generate-coverage: true

```

Reply:
Think we'd have to create variations of the workflow to support use of services.